docs/checkers/index.md

```diff
@@ -400,6 +400,25 @@ if True:  # Error was on this line
   pass
 ```
 
+### Use list literal (R1734) [](#R1734)
+
+This error occurs when list() is used instead of [] to create an empty list.
+
+```{literalinclude} /../examples/pylint/R1734_use_list_literal.py
+
+```
+
+The above can be modified to:
+
+```python
+lst = [1, 2, 3, 4]
+even_lst = [] # Error on this line.
```

Comment:
Don't include the "Error of this line" comment. Instead, modify the comment to indicate that this is the fixed version.

Also, for comments please make sure to use two blank spaces before the `#`, not one.